src/ui/controls/RadioListControl.tsx

```diff
@@ -40,6 +40,15 @@ export class RadioListControl extends React.Component<IStringControlProps, void>
   }
 
   /** @override */
```

Comment:
But it's not overriding anything:

https://github.com/facebook/react/blob/c78464f8ea9a5b00ec80252d20a71a1482210e57/src/isomorphic/modern/class/ReactComponent.js

https://github.com/facebook/react/blob/3bc5595dfd6db741b19f17e7aef98b2c1e4be847/src/renderers/shared/stack/reconciler/ReactCompositeComponent.js#L331